plugin/documents.py

```diff
@@ -152,6 +158,17 @@ def __del__(self) -> None:
 
     # --- Implements AbstractViewListener ------------------------------------------------------------------------------
 
+    def on_pre_move_window_async(self) -> None:
+        if self._registered and self._manager:
+            self._manager.unregister_listener_async(self)
+            self._registered = False
+            self._manager = None
+        self._cleanup()
+
+    def on_post_move_window_async(self) -> None:
+        self._setup()
+        self.on_activated_async()
```

Comment:
It appears that if I drag a tab to a different position within the same window then LSP stops working for that view (no LSP messages trigger anymore).

Reply:
this should work now